lesson6/index.md

```diff
@@ -46,222 +49,80 @@ img {
+## CSS Fonts
 
-Second, one element can have multiple classes by separating them with a space, but an element can _not_ have more than one ID.
+A crucial role on all websites plays the font. It can give a totally different look and feel on your website.
 
-This is fine:
+This section will cover:
 
-```html
-<h1 class="red-heading large-text">
-  I want this heading to be red and have large text!
-</h1>
-<h1 class="red-heading">Another red heading!</h1>
-<h1 class="blue-heading">And I want this heading to be blue!</h1>
-<h1 class="blue-heading full-width">
-  This one too! And I want it to be extra wide!
-</h1>
-<h1 class="red-heading">Back to red!</h1>
-<h1 class="blue-heading">Back to blue!</h1>
-```
+- Font-families
+- Font-style [`font-style`](https://www.w3schools.com/css/css_font_style.asp)
 
-Do **not** do this, it is not valid HTML because two elements have the same `id` attribute:
+## CSS Basic Colors
 
-```html
-<!-- Don't do this! -->
-<h1 id="red-heading">I want this heading to be red!</h1>
-<h1 id="red-heading">I should have used a class instead...</h1>
-<h1 id="blue-heading small-text">I should have used a class instead...</h1>
-<!-- Don't do this! -->
-```
+This section will cover some basic CSS colors [standard color names](https://www.w3schools.com/colors/colors_names.asp) (e.g. Tomato, Azure, DodgerBlue, etc.). It will also practice the usage of those color names together with the following CSS Rules, which students will learn:
 
-It is very useful to be able to have multiple elements with the same `class` attribute. That way, you can apply the same style to a whole group of elements at the same time!
+- Background Color
+- Text Color
+- Border Color (keep in mind that this only covers the border color without the border style and sizes. We can use `2px solid` without going into details)
 
-And adding multiple classes to an element means you can make your classes composable, meaning you can combine several classes to achieve a desired result (such as having separate `red-heading` and `large-text` classes, one of which sets a color and one of which sets a font-size).
+# Exercise Description
 
-So what is the point of IDs if they are not reusable?
+1. Practice using CSS styles with this [codepen](https://codepen.io/redi-school/pen/bGxywzN)
 
-Well, they are often used for Javascript or for linking to a specific part of a page. And sometimes they are useful for applying CSS styles as well, especially when chaining selectors. More on that topic in following lessons!
+2. Create your own personal biography page, including:
 
-## Exercise 1
+- multiple different headings,
+- multiple paragraphs (you can use lorem\*5 to quickly fill with placeholder text)
+- lists,
+- images
+  Ad CSS styles to your page. Try to use everything we learned today!
 
-Fork this CodePen and follow the instructions to add styles using class and ID selectors: [https://codepen.io/redi-school/pen/poOMMJx](https://codepen.io/redi-school/pen/poOMMJx)
```

Comment:
Could you please include short recap from the previous class how to use classes? We started onsite with it, but it would be good to remind students about class selector the syntax

Reply:
Sorry, I forgot to add you Synthia!
We decided in the beginning of semester to teach this topic in the next sesson. Class and ID selectors will be discussed on lesson 7.
The order of the topics changed

Reply:
No problem Joana! I saw the course plan has changed, the next lesson is about Units (px/%), CSS text formatting, borders & background colours.
I'm not suggesting to change the course but do homework check and maybe spend 5 to 10min at the beginning for the recap in order to use these CSS attributes together with CSS class selectors that they learned last time. It's a public holiday tomorrow and they just started with CSS- repetition is a good practice in continuous learning!🙂

Reply:
ahh, good suggestion :) Thank you Syntia, I will update it!

Reply:
Thanks!